model_lab_configs/huggingface/microsoft/Phi-3.5-mini-instruct/1/infra/provision/finetuning.parameters.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
I would suggest they work like an add-on. Only apply to the model project when user enable it